docs/docs/transpiler.mdx

```diff
@@ -112,7 +112,7 @@ You can also provide a custom transpiler by creating a class that implements the
 import { TranslocoTranspiler } from '@ngneat/transloco';
 
 export class CustomTranspiler implements TranslocoTranspiler {
-  transpile(value: any, params, translation: Translation) {
+  transpile(value: any, params, translation: Translation, key?: any) {
```

Comment:
Why is this optional and typed as any?

Reply:
Updated to non-optional. 

More discussion here: https://github.com/ngneat/transloco/pull/569#discussion_r887043400